contributors/guide/README.md

```diff
@@ -5,100 +5,174 @@ aliases: [ "/guide" ]
 ## Setting up your development environment
 
-If you haven’t set up your environment, check the [developer resources](/contributors/devel/README.md#setting-up-your-dev-environment-coding-and-debugging).
+It is not required to set up a developer environment in order to contribute to Kubernetes.
+
+If you plan to contribute code changes, 
+review the [developer resources]
+page for how to set up your environment.
 
 ## Community Expectations and Roles
 
 Kubernetes is a community project.
-Consequently, it is wholly dependent on its community to provide a productive, friendly and collaborative environment.
+Consequently, it is wholly dependent on its community to provide a productive, 
+friendly and collaborative environment.
 
-- Read and review the [Community Expectations](expectations.md) for an understanding of code and review expectations.
-- See [Community Membership](/community-membership.md) for a list the various responsibilities of contributor roles. You are encouraged to move up this contributor ladder as you gain experience.
+- Read and review the [Community Expectations] for an 
+understanding of code and review expectations.
+- See [Community Membership] for a list the various 
+responsibilities of contributor roles. 
+- You are encouraged to move up this contributor ladder as you gain experience.
 
-# Kubernetes Contributor Playground
+## Kubernetes Contributor Playground
 
-If you are looking for a safe place, where you can familiarize yourself with (some of) the Kubernetes Project's review and pull request processes, then the [Kubernetes Contributor Playground](https://github.com/kubernetes-sigs/contributor-playground/) is the right place for you.
+If you are looking for a safe place, where you can familiarize yourself with 
+the pull request and issue review process 
+in Kubernetes, then the [Kubernetes Contributor Playground]
+is the right place for you.
 
 ## Contributor Workshops
 
-A [Youtube playlist](https://www.youtube.com/playlist?list=PL69nYSiGNLP3M5X7stuD7N4r3uP2PZQUx) of the New Contributor workshop has been posted, and an outline of the video content can be found [here](/events/2018/05-contributor-summit). 
+A [Youtube playlist]
+of the New Contributor workshop has been posted.
+An outline of the video content can be found [here]. 
 
-# Community
+## Community
 
-If you haven't noticed by now, we have a large, lively, and friendly open-source community.
-We depend on new people becoming members and regular code contributors, so we would like you to come join us!
-The [Community Membership Document](/community-membership.md) covers membership processes and roles.
+Kubernetes is a large, lively, friendly open-source community.
+As many open source projects often do, 
+it depends on new people becoming members and regular code contributors. 
+The [Community Membership Document] 
+covers membership processes and roles. 
+Please consider joining Kubernetes, and making your way up the contributor ladder!
 
 ## Communication
```

Comment:
```suggestion
### Communication
```
The items here through Mentoring I think are intended to be subheadings under community.